setup.py

```diff
@@ -213,7 +213,7 @@ def run(self):
     'pysha3==0.3',
     'pytest==2.9.2',
     'pytest-cov==2.2.1',
-    'python-bitcoinlib==0.7.0',
+    'python-bitcoinlib', #==0.7.0
```

Comment:
with new installs it will use the newest version on pypi (0.7.0). with existing installs it won't enforce a particular version. I tried a few other version strings like `'python-bitcoinlib==0.7.0, ==0.7.1-SNAPSHOT',` and `'python-bitcoinlib>=0.7.0',` with no luck, but let me try again with this new code as I agree this needs to change